scheduling-manager-workflows/src/main/java/com/amazonaws/blox/schedulingmanager/deployment/steps/GetDeploymentData.java

```diff
@@ -36,7 +36,15 @@ public void handleRequest(InputStream input, OutputStream output, Context contex
       throws IOException {
     log.debug("getDeploymentData lambda");
 
-    final DeploymentInput deploymentInput = encoder.decode(input, DeploymentInput.class);
+    final DeploymentInput deploymentInput;
+    try {
+      deploymentInput = encoder.decode(input, DeploymentInput.class);
+    } catch (final IOException e) {
```

Comment:
Might make sense to move this repeated stanza into the encoder.

Reply:
good idea